index.html

```diff
@@ -1,175 +1,238 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rajvardhan Portfolio</title>
-    <link rel="stylesheet" href="index.css">
+    <!-- <link rel="stylesheet" href="index.css"> -->
     <script src="https://kit.fontawesome.com/1b30b8cf7f.js" crossorigin="anonymous"></script>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
 
-    <!-- <link href="./output.css" rel="stylesheet"> -->
+    <link href="./output.css" rel="stylesheet">
 </head>
-<body>
-<div id="header">
-    <div class="container">
-        <nav>
-            <img src="images/RAJVARDHAN (2).png" class="logo" alt="Logo img">
-            <ul id="sidemenu">
-                <li><a href="#header">Home</a></li>
-                <li><a href="#about">About</a></li>
-                <li><a href="#services">Services</a></li>
-                <li><a href="#portfolio">Portfolio</a></li>
-                <li><a href="#contact">Contact</a></li>
-                <i class="fa-solid fa-xmark" onclick="closemenu()"></i>
-            </ul>
-            <i class="fa-solid fa-bars" onclick="openmenu()"></i>
-        </nav>
-        <div class="header-text">
-            <p>CSE Student</p>
-            <h1 class="text-3xl font-bold underline">Hi, I'm <span>Rajvardhan</span> <br> Patil from KITCoEK</h1>
-        </div>
-    </div>
-</div>
 
-<!-- ____________about______________ -->
-<div id="about">
-    <div class="container">
-        <div class="row">
-            <div class="about-col-1">
-                <img src="images/IMG_20240517_125117.jpg" alt="profile image">
+<body class="bg-[#080808] text-white font-['Poppins'] scroll-smooth">
+    <div id="header" class="w-full h-screen bg-cover bg-center bg-bgimg1 md:bg-bgimg2">
```

Comment:
if we use the right class name in the tailwind config, we don't need both the classes